homeassistant/util/loop.py

```diff
@@ -126,6 +126,9 @@ def _dev_help_message(what: str) -> str:
     )
 
 
+_depth = [0]
```

Comment:
`linecache` can potentially does a blocking `stat` which would cause a loop which is why we have a depth check.  There is currently no solution for `linecache` at this time